smart_shepherd/ml_api/app/utils/preprocessor.py

```diff
@@ -27,11 +27,11 @@ def temperature_parser(self) -> float:
 
 		orion = config.URL_WEATHER_ORION
 		sensor_id = config.TEMPERATURE_SENSOR_ID
-		url = f"http://{orion}/ngsi-ld/v1/entities/urn:ngsi-ld:TemperatureSensor:{sensor_id}"
-		headers = {"Content-Type": "application/json"}
+		url = f"http://{orion}:1026/ngsi-ld/v1/entities/urn:ngsi-ld:TemperatureSensor:{sensor_id}"
+		header = config.HEADER
```

Comment:
Why do you take the header from a config Is there any case where it can change?

Reply:
I don't know exactly which case, but I thought it could change or a new field in the headers dict could be added in case needed.

Reply:
I would make that a constant. The headers are defined by the api. Unless the api changes, they will stay stable. There some where you would want to make the value configurable(f.e. the tenant or the link header), but that not the case here. Only makes the code more confusing